### PR TITLE
[45] product Page Using HTML

### PR DESCRIPTION
**Pull request by Moronke02**
## Task: `Develop  a product page for your e-commerce store using HTML only`
- `Added HTML folder`
-Link to implementation screenshots:
 https://drive.google.com/drive/folders/1P7JaWxvvj2fW4ymBTuyMSqs3ufECf8rv?usp=sharing

### CODE REVIEW

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Moronke02 Your index.html should also be the product page. You need to remove the index file. 
You are meant to only write html no styling, you need to remove all styling in the product page alongside the sizing of the images. 
You also need to remove all images that you are not using from the asset folder.

**Comment by Moronke02** [comment]
> @Moronke02 Your index.html should also be the product page. You need to remove the index file.
> You are meant to only write html no styling, you need to remove all styling in the product page alongside the sizing of the images.
> You also need to remove all images that you are not using from the asset folder.

@ToluAkin Thanks for the feedback. Changes effected.

**Comment by ToluAkin** [comment]
@Moronke02 you are meant to upload the screenshot of the webpage on the pull request.

**Comment by ToluAkin** [comment]
@Moronke02 You should have your image names in lowercase with no spaces between them. You are also meant to work with two product images

**Comment by Moronke02** [comment]
> @Moronke02 you are meant to upload the screenshot of the webpage on the pull request.

@ToluAkin It was stated that we can add the link to the screenshot although if that is most expected I think I will just do that

**Comment by Moronke02** [comment]
> @Moronke02 You should have your image names in lowercase with no spaces between them. You are also meant to work with two product images

@ToluAkin Thanks for the feedback, I will correct them as you said . But for the product images, it was not stated although I will also eliminate the redundant ones.Thanks

**Comment by Moronke02** [comment]
Implementation Screenshots
Screenshot (315)
Screenshot (316)

File: submissions/moronkeAliu/HTML/index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                <h4>Rainbow Fruit Parfait</h4>
+                <p class="price">#2000</p>
+                <p><em>A colorful and great way of starting a<br> healthy day with a cup of parfait <br> is all you need!</em></p>
+                <p><button>Order Now</button></p>
+              </div>
+            </div>
+          </div>
+       
+    </body>
+    <footer>
```
Comment (adaugochi) [inline submissions/moronkeAliu/HTML/index.html:83]: Footer tag should be inside your <body> tag

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**